### PR TITLE
add exploitation subsection

### PR DESCRIPTION
**Pull request by minrk**

### CODE REVIEW

File: impact.tex [hunk 1]
```diff
@@ -460,55 +460,68 @@ \subsubsection{Dissemination of results}
...
+
 
 These resources will be hosted on free, public hosting services,
-such as GitHub Pages, YouTube channels and as much as possible co-developed and co-hosted with existing and 
+such as GitHub Pages, YouTube channels and as much as possible co-developed and co-hosted with existing and
 well-established organizations
 (The Carpentries, CodeRefinery, Galaxy Training Network, Pangeo, etc.).
 
 
 The operation of prototype deployments of binder services
```
Comment (fangohr) [inline impact.tex]: This is a duplicated paragraph (see next comment). Any preference which of the 2 nearly identical versions you'd like to keep?

File: impact.tex [hunk 2]
```diff
@@ -460,55 +460,68 @@ \subsubsection{Dissemination of results}
...
+and sites expecting publications have budgeted funds for paying Open Access fees.
+We will identify and attend appropriate conferences for disseminating our work,
+including running tutorials at conferences in historically interested communities such as PyData and SciPy.
+Also, we will identify and attend conferences from complementary communities such as ROpenSci,
+Mozilla Science, and Julia
+as well as domain specific conferences to maximise the impact of \TheProject and to broaden its
+audience outside the
+traditionally included communities.
+
+The operation of prototype services in \WPref{applications} is also a dissemination activity,
```
Comment (fangohr) [inline impact.tex:495]: Duplicated paragraph (second copy)

**Review by fangohr — COMMENTED** [review]
(Review still ongoing.)

**Comment by fangohr** [comment]
The data management plan is part of the 'dissemination'. Is that on purpose? I would have taken, to have the 3 'official' D&E&C together. For example:

- dissemination
- exploitation
- communication
- data management plan

File: impact.tex (hunk 2 above)
Comment (minrk) [reply impact.tex:495]: Thanks, botched merge.

**Comment by fangohr** [comment]
A pleasure to read.

File: impact.tex (hunk 2 above)
Comment (minrk) [reply impact.tex:495]: Let's keep the later one

**Pull request merged**